README.md

```diff
@@ -148,14 +148,27 @@ $ minikube start --vm-driver kvm2 --feature-gates=DevicePlugins=true --memory 40
 
 ```bash
 oc cluster up --skip-registry-check --enable=router,sample-templates
+oc cluster down
+sed -i "/kind/ a\kubeletArguments:\n  feature-gates:\n  - DevicePlugins=true" $PWD/openshift.local.clusterup/node/node-config.yaml
+oc cluster up --skip-registry-check
 ```
 
-Apply the following workaround:
+### Running on `Minishift`
+
+Update the node-config.yaml
+
+```bash
+sudo sed -i "/kind/ a\kubeletArguments:\n  feature-gates:\n  - DevicePlugins=true" /var/lib/minishift/base/node/node-config.yaml
+```
+
+NOTE: For Minishift, updation of node-config.yaml file and device plugins fix should be done by logging into minishift vm via `minishift ssh`.
+
+Apply the following workaround for both Origin and Minishift:
 
 ```bash
 # Fix device plugins
 # Workaround for https://github.com/openshift/origin/pull/20351
-KUBELET_ROOTFS=$(sudo docker inspect $(sudo docker ps | grep kubelet | cut -d" " -f1) | jq -r ".[0].GraphDriver.Data.MergedDir" -)
```

Comment:
Used a common command which would be useful for both `origin` and `Minishift`, as `jq` package would be required to install on `Minishift`.